packages/client.api/src/OsdkObjectFrom.ts

```diff
@@ -139,7 +140,7 @@ export type Osdk<
     /** @deprecated use $primaryKey */
     __primaryKey: Q extends ObjectTypeDefinition<any>
       ? OsdkObjectPrimaryKeyType<Q>
-      : unknown;
+      : any;
```

Comment:
Had to type this as any because unknown doesn't work when we have an interface type. It used to work when `PropertyValueClientToWire` basically also evaluated to unknown, so it'd be an unknown to unknown check, but now its unknown to `string|boolean|number|attachment...`